Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -153,17 +163,35 @@ bool
 BTDiagnostics::DoComputeAndPack (int step, bool force_flush)
 {
     // always set to true for BTDiagnostics since back-transform buffers are potentially
-    // computed and packed every timstep.
-    return true;
+    // computed and packed every timstep, except at initialization when step == -1.
+    if (step>=0) {
+        return true;
+    }
+    return false;
 }
```

Comment:
If the short version in the .H file works, this could be removed.

Reply:
It was leading to compiler error. I reverted the implementation to cpp, but with return(step>=0)